bank_integration/bank_integration/api/hdfc_bank_api.py

```
@@ -782,8 +789,23 @@ def payment_success(self):
         payment_entry_doc = frappe.get_doc("Payment Entry", self.data.docname)
         payment_entry_doc.online_payment_status = "Paid"
         payment_entry_doc.reference_no = ref_no
-        payment_entry_doc.submit()
-        frappe.db.commit()
+        try:
+            payment_entry_doc.submit()
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)